.env

```diff
@@ -0,0 +1,4 @@
+USER='sergey'
```

Comment:
This file is **only** yours. it's your personal config, so it should not be in git.
For that reason there is [.gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore) file, use it.

Reply:
1. remember this file content
2. `git rm .env`
3. make new commit